ext/jsonnet/callbacks.c

```diff
@@ -123,14 +129,26 @@ import_callback_entrypoint(void *ctx, const char *base, const char *rel, char **
 
     if (state) {
 	VALUE msg = rescue_callback(state, "cannot import %s from %s", rel, base);
+#ifdef HAVE_JSONNET_IMPORT_CALLBACK_0_19
+	*buf = rubyjsonnet_str_to_cstr(vm->vm, msg);
+	*buflen = RSTRING_LEN(msg);
+	return 1;
+#else
 	*success = 0;
 	return rubyjsonnet_str_to_cstr(vm->vm, msg);
+#endif
     }
 
     result = rb_Array(result);
-    *success = 1;
     *found_here = rubyjsonnet_str_to_cstr(vm->vm, rb_ary_entry(result, 1));
+#ifdef HAVE_JSONNET_IMPORT_CALLBACK_0_19
+    *buf = rubyjsonnet_str_to_cstr(vm->vm, rb_ary_entry(result, 0));
+    *buflen = RSTRING_LEN(rb_ary_entry(result, 0));
```

Comment:
`rubyjsonnet_str_to_cstr()` calls `#to_str` on a ruby object if defined, but an original ruby object is passed to RSTRING_LEN here. I guess one option would be something like

```c
    VALUE content = rb_ary_entry(result, 0);
    StringValue(content);
    *buf = rubyjsonnet_str_to_ptr(vm->vm, content);
    *buflen = RSTRING_LEN(content);
```

In this case `rubyjsonnet_str_to_ptr()` can call `RSTRING_PTR` instead of `StringValuePtr`.

Reply:
c.f. https://github.com/yugui/ruby-jsonnet/commit/41c9505e514267fda1392a90e5403fa6e7f77edb

Reply:
I think that's great, thank you  :+1: